suite-common/redux-utils/src/extraDependenciesType.ts

```diff
@@ -27,7 +27,7 @@ export type ExtraDependencies = {
             timestamp: number;
         }>;
         cardanoFetchTrezorPools: SuiteCompatibleThunk<'tADA' | 'ADA'>;
-        initMetadata: SuiteCompatibleThunk<boolean>;
+        initMetadata: SuiteCompatibleThunk<undefined>;
```

Comment:
nitpick: if you use void instead of undefined you don't need to pass undefined in discoveryThunks...
```suggestion
        initMetadata: SuiteCompatibleThunk<void>;
```

Reply:
in [f228f6b](https://github.com/trezor/trezor-suite/pull/9130/commits/f228f6be38ab14be056817b6decbfc54721ae20c) I re-introduced booelan param. I believe it is still useful. Sometimes we want to init metadata on click even if it is disabled or cancelled on device. And sometimes we want to respect that metadata is disabled. thus force param to init